protocols/horizon/main.go

```diff
@@ -451,3 +451,43 @@ func MustKeyTypeFromAddress(address string) string {
 
 	return ret
 }
+
+type AggregateBalanceRecord struct {
+	Id               string `json:"id"`
```

Comment:
from what I gather, "id" is used to identify the page returned. in some cases, its the account id but in others, it might a tx hash or another identifier for the page. In other words, I'll need to use account_id instead of id.